libsignal-service/src/push_service.rs

```diff
@@ -137,6 +172,25 @@ pub trait PushService {
         Ok(SmsVerificationCodeResponse::SmsSent)
     }
 
+    async fn request_voice_verification_code(
+        &mut self,
+    ) -> Result<VoiceVerificationCodeResponse, ServiceError> {
+        self.get(CREATE_ACCOUNT_VOICE_PATH).await?;
+        Ok(VoiceVerificationCodeResponse::CallIssued)
+    }
+
+    async fn confirm_device(
+        &mut self,
+        confirm_code: u32,
+        confirm_code_message: &ConfirmCodeMessage,
+    ) -> Result<DeviceId, ServiceError> {
+        self.put(
+            dbg!(&format!("{}{}", DEVICE_PATH, confirm_code)),
```

Comment:
Stale `dbg!` (didn't  know about its existence, TIL), does `log` have an equivalent?

Reply:
it's really not important, I'll just remove it.